src/app/common/common.tsx

```diff
@@ -868,7 +868,12 @@ class Markdown extends React.Component<
     {}
 > {
     CodeBlockRenderer(props: any, codeSelect: boolean, codeSelectIndex: number): any {
-        let codeText = codeSelect ? props.node.children[0].children[0].value : props.children;
+        let codeText:string ="";
+        try {
+            codeText = props.node.children[0].children[0].value;
```

Comment:
This should never fail, and it never has in practice, but I don't trust my understanding of the DOM well enough to be sure of this 
This try catch just makes sure it fails silently if any of the things I'm referencing don't exist, like node, or either of the children, or value 

is it okay to use try catch here?

Reply:
Should I do some kind of error propagating? I think it's better if it fails silently if something bad happens. 
Because the markdown would still show, but the only thing that would break is the ability to copy the markdown
(note that in tracy's error case, this will no longer break, so copying will work as intended)